### PR TITLE
#5404 [Manager] Replace singular resource with plural in inspect api

### PR DESCRIPTION
**Pull request by saivirtue**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [x] Add unit tests (or please explain why this PR is not worth having new tests)
Only change words
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/cypress/it/paper/propertyView.test.ts [hunk 1]
```diff
@@ -390,12 +390,14 @@ describe('Property view', () => {
...
-      .parent('.topic')
-      .find('.topic-status.running')
-      .should('exist'); // Status
+    cy.get('#paper').within(() => {
+      // topic "T1" will be auto created
+      cy.findByText('T1')
+        .should('exist')
+        .parent('.topic')
+        .find('.topic-status.running')
+        .should('exist'); // Status
```
Comment (konekoya) [inline ohara-manager/client/cypress/it/paper/propertyView.test.ts:399]: Sorry about the trouble 😅

Comment (saivirtue) [reply ohara-manager/client/cypress/it/paper/propertyView.test.ts:399]: No worries :)

**Comment by saivirtue** [comment]
retry manager-api

**Comment by chia7712** [comment]
thanks for this quick migrating!!!

**Pull request merged**